functest/test_utils.go

```diff
@@ -44,12 +44,12 @@ import (
 	"github.com/pkg/errors"
 )
 
-type contractInfo struct {
+type ContractInfo struct {
```

Comment:
Why you make it public? I don't see any reason.

Reply:
utils_test.go used to be test class, though it has no any test methods. I renamed it in order to make its methods and variables public, this change makes  ContractInfo visible from main_test_wrapper.go

Reply:
Just revert these changes, please